src/main.cpp

```diff
@@ -100,6 +104,8 @@ int main(int argc, char *argv[])
     }
 
     std::vector<unsigned char> bytecode;
+    // TODO - Allow reading debug information from file.
+    DebugInfo *debug = nullptr;
```

Comment:
Well, the DebugInfo has an explicit constructor, and no = operator, and no initialization method, so it really has to be created by an explicit call to the constructor, but we need to use it outside of the scope it's CREATED in, so, I figured the best way to do this is with a pointer...